### PR TITLE
Update Manual Test - Node Delete

### PR DESCRIPTION
**Pull request by c3y1huang**
https://github.com/longhorn/longhorn/issues/2462

### CODE REVIEW

File: docs/content/manual/release-specific/v1.1.1/delete-node.md [hunk 1]
```diff
@@ -25,15 +42,17 @@ kubectl create -f https://raw.githubusercontent.com/longhorn/longhorn/v1.1.0/exa
...
+*And* should see node-1 removed from the node list in the browser.
 
 *And* should see node-1 removed from nodes.longhorn.io.
 ```
 kubectl -n longhorn-system get nodes.longhorn.io
 ```
 
 
-## Test Node Delete - should fail when volume replica not evicted on node
+## Test Node Delete - when node schedule is enabled and not evicted
```
Comment (innobead) [inline docs/content/manual/release-specific/v1.1.1/delete-node.md:55]:
```suggestion
## Test Node Delete - should delete the node when the node is down, even the schedule is enabled and replicas not evicted from the node
```

**Review by innobead — CHANGES_REQUESTED** [review]
LGTM. Thanks @c3y1huang proactively working on this :+1: 

@khushboo-rancher please take a look. thanks.

File: docs/content/manual/release-specific/v1.1.1/delete-node.md (hunk 1 above)
Comment (c3y1huang) [reply docs/content/manual/release-specific/v1.1.1/delete-node.md]: Addressed.

**Review by innobead — DISMISSED** [review]

File: docs/content/manual/release-specific/v1.1.1/delete-node.md [hunk 2]
```diff
@@ -25,15 +42,17 @@ kubectl create -f https://raw.githubusercontent.com/longhorn/longhorn/v1.1.0/exa
...
+*And* should see node-1 removed from the node list in the browser.
 
 *And* should see node-1 removed from nodes.longhorn.io.
 ```
 kubectl -n longhorn-system get nodes.longhorn.io
 ```
 
 
-## Test Node Delete - should fail when volume replica not evicted on node
+## Test Node Delete - should delete the node when the node is down, even the schedule is enabled and replicas not evicted from the node
```
Comment (khushboo-rancher) [inline docs/content/manual/release-specific/v1.1.1/delete-node.md:55]: If the a replica exists on the node and the volume is detached. In that case Longhorn fails to remove the replica and finally fails to remove the node. 
Should we make a note about it?

**Review by khushboo-rancher — DISMISSED** [review]
LGTM, just one suggestion to add a note.

File: docs/content/manual/release-specific/v1.1.1/delete-node.md (hunk 2 above)
Comment (c3y1huang) [reply docs/content/manual/release-specific/v1.1.1/delete-node.md:55]: From the test, the volume became degraded and longhorn will remove the node and the replica. I've updated the commit by having more validation steps and examples.

Comment (innobead) [reply docs/content/manual/release-specific/v1.1.1/delete-node.md:55]: Like we discussed, we probably still need to make note for the 1 replica detached volume case.

Comment (c3y1huang) [reply docs/content/manual/release-specific/v1.1.1/delete-node.md:55]: Thanks @innobead , I've missed the case for 1 replica.

Comment (c3y1huang) [reply docs/content/manual/release-specific/v1.1.1/delete-node.md:55]: Added.

**Review by innobead — APPROVED** [review]

**Pull request merged**